compiler-plugin-tests/src/test/resources/docker-cloud-test/project-based-tests/cloud-flag-code-coverage/tests/some_tests.bal

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+@test:Config{}
+function smallTest() {
+    test:assertEquals(1, 1, "1 should be equal to 1");
+}
+
+@test:Config{}
+function anotherSmallTest() {
+    test:assertEquals(2, 2, "2 should be equal to 2");
+}
+
+@test:Config{}
+function anotherSmallTest2() {
+    test:assertNotEquals(2, 3, "2 should not be equal to 3");
+}
```

Comment:
Add new line

Reply:
Added in [5fc44be](https://github.com/ballerina-platform/module-ballerina-c2c/pull/765/commits/5fc44befe4d898c702ef885cea29fd93df2f73cb)